package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@stitch-fix/log-weasel",
+  "description": "Log Weasel Javascript client",
+  "version": "0.0.5",
+  "author": "Stitch Fix",
+  "license": "UNLICENSED",
+  "main": "src/index.js",
+  "repository": "https://github.com/stitchfix/log_weasel",
+  "jest": {
+    "testURL": "http://localhost"
+  },
+  "scripts": {
+    "clean": "rm build/app.js",
+    "test": "jest",
+    "lint": "eslint --ext .js --ext .jsx .",
```

Comment:
The react community no longer uses JSX -- just js for files with JSX in them. Want to change all files to JS in this repo?

Reply:
Ah whoops - meant to remove this. Will do!